ops/ansible/roles/postmaster_deploy/tasks/mysql.yml

```diff
@@ -13,10 +13,6 @@
   become: yes
   apt: pkg=mysql-server state=latest
 
-- name: Install mysql-python globally for this next command :(
```

Comment:
@philof, since Ansible needs this, can you install `python-mysql` using apt? I think that's the cleaner way to go instead of using pip.